xinference/model/llm/llm_family.json

```diff
@@ -421,16 +432,26 @@
         ],
         "model_id": "Xorbits/chatglm2-6B-GGML",
         "model_file_name_template": "chatglm2-ggml-{quantization}.bin"
+      },
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 6,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_id": "THUDM/chatglm2-6b-32k"
```

Comment:
Better create a new model family since chatglm-32k is actually a different model.